crates/turbopack-css/src/asset.rs

```diff
@@ -189,6 +192,10 @@ impl CssChunkItem for ModuleChunkItem {
         let mut visitors = Vec::new();
         let mut root_visitors = Vec::new();
         for code_gen in code_gens {
+            for import in &code_gen.imports {
+                imports.push(import.clone());
+            }
```

Comment:
I think we can use extend to combine into a single reallocation?
```suggestion
            imports.extend(code_gen.imports.clone());
```